src/flags.ts

```diff
@@ -60,6 +67,7 @@ const allFlags: FlagMetadata[] = [
   { name: "noLang", defaultOnStages: [] },
   { name: "translate", defaultOnStages: [] },
   { name: "noWelcome", defaultOnStages: ["local", "REVIEW"] },
+  { name: "pwa", defaultOnStages: ["REVIEW"] },
```

Comment:
This needs more thought/discussion. If you install the PWA, close it, then reopen it, you'll lose the flag and the ability to add the flag. In the meantime, this change will make the review branch a bit more useful for testing.